bundler/lib/dependabot/bundler/update_checker/latest_version_finder.rb

```diff
@@ -93,71 +92,6 @@ def filter_lower_versions(versions_array)
             select { |version| version > Gem::Version.new(dependency.version) }
         end
 
-        def registry_versions
```

Comment:
This is the main method I wanted to sink into a separate class as it requires strong internal contract on `dependency_source` that will change dramatically